src/schema/workflow_execution_activity.yaml

```diff
@@ -226,6 +226,14 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:wfmb-{id_shoulder}-{id_blade}{id_version}{id_locus}"
           interpolated: true
+      git_url:
+        description: >-
+          The url that points to the github location of the metabolomics workflow.
+        comments: >-
+          The git_url must point to an exact workflow if multiple workflows share the
```

Comment:
This is good guidance, but I would prefer not to be customizing slots so much To what degree can we generalize this rule over the whole schema?

Reply:
Please give some examples of exact an inexact workflows. if you can't think of a  good place to put that in the schema, just paste them into comments here.

Reply:
@kheal  can you give an example of these exact and inexact workflows. I tried to clarify in the `comments`. @turbomam  are you saying you would rather add this comment to the global definition of `git_url`?

Reply:
Are you comfortable checking for patterns in existing data to look for patterns?

Can I see some source data (that may not be loaded in MongoDB yet?) that shows where things have been done with good or insufficient exactness?

Reply:
> are you saying you would rather add this comment to the global definition of `git_url`

I want us to consider that kind of thing any time we make any change to the schema, especially `slot_usage`s, `rules`, etc.

THis kind of request should always be supported by data, either in the schema, as part of the PR, or in example files.

Reply:
EMSL people are the main contributors to the schema nowadays, but I am concerned that the schema is being disproportionately customized for spectromety assays and workflows. It's especially difficult because I'm not able to see the data or the code.

Reply:
Currently the github repo [MetaMS](https://github.com/microbiomedata/metaMS) houses the GC-MS metabolomics workflow.  In the (near) future, it will also house the LC-MS lipidomics workflow and (further in the future) the LC-MS metabolomics workflows.  So pointing to `https://github.com/microbiomedata/metaMS` is insufficient, since these are all examples of workflows used for `MetabolomicsAnalysis`.  Instead, we should point to this url to specify which workflow from this repo as used `https://github.com/microbiomedata/metaMS/blob/master/metaMS/gcmsWorkflow.py`.

Reply:
@turbomam I added the examples Katherine gave to the comments

Reply:
Thanks. I wonder if we should use an enumeration here? I can't remember if switching from a string range in the global slot definition to an enumeration in a slot usage is considered monotonic.

Reply:
Can one of you please check with some non-EMSL people about their practices for populating the `git_url` slot? Maybe @scanon ?

Reply:
I am thinking we would want to leave it as a string to make space for other git_urls that come from outside of emsl. As you mentioned, you are worried about making the schema to specific to emsl - I think make this an enumeration would create too much specificity.